README.md

```diff
@@ -26,14 +23,19 @@ Use the following HTML to get a full-width FT affiliation ribbon.  Ideally this
 			<a class="o-ft-affiliate-ribbon__logo" href="https://www.ft.com/" title="The Financial Times" target="_blank">
 				<span class="o-ft-affiliate-ribbon__visually-hidden">Financial Times</span>
 			</a>
-		</div>	
+		</div>
 	</div>
 </div>
 ```
 
 ### Sass
 
-As with all Origami components, o-ft-affiliate-ribbon has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). To use its compiled CSS (rather than using its mixins with your own Sass) set `$o-o-ft-affiliate-ribbon-is-silent : false;` in your Sass after you've imported the o-ft-affiliate-ribbon Sass.
+As with all Origami components, o-ft-affiliate-ribbon has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). To use its compiled CSS (rather than using its mixins with your own Sass) set `$o-o-ft-affiliate-ribbon-is-silent : false;` in your Sass after you've imported the o-ft-affiliate-ribbon Sass:
```

Comment:
"after" should be "before" in the line `in your Sass after you've imported`

Also, the variable has a double-o prefix `$o-o-`